dict/dict.go

```diff
@@ -0,0 +1,80 @@
+package dict
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+)
+
+func GetDnToAciClassMap(childClass string, parentPrefix string) string {
+	rnMapping := make(map[string]map[string]string)
+
+	resp, err := http.Get("https://pubhub.devnetcloud.com/media/model-doc-latest/docs/doc/jsonmeta/aci-meta.json")
```

Comment:
So every time this function is called we are retrieving the full meta? 

Should we do this one time and cache it in memory or store it as static file in the repo? 

As static file we might want because what happens when the meta from devnet deviates from the meta information we are using in the generator? Other option is to create a template in the generator that construct the information you need here.